src/components/Editor/Organisations/LinkOverlay.vue

```diff
@@ -528,17 +533,29 @@
         }
       },
       watch: {
-        'menus.newOrganisation.data.logo': function(){
-          if (this.menus.newOrganisation.data.logo) {
-            let reader = new FileReader();
-            reader.addEventListener('load', async (event) => {
-              this.menus.newOrganisation.logoData = {
-                filename: this.menus.newOrganisation.data.logo.name,
-                data: event.target['result'],
-                content_type: this.menus.newOrganisation.data.logo.type
-              };
-            });
-            reader.readAsDataURL(this.menus.newOrganisation.data.logo);
+        'menus.newOrganisation.data.logo': {
```

Comment:
This watch could be removed.

Reply:
On trying to remove this I see why it was originally done; if I attempt to access `this.menus.newOrganisation.data` within `createNewOrganisation` then it's always equal to `{}` (not so if it's done in the watcher).